pages/setup/account.page.test.tsx

```diff
@@ -0,0 +1,134 @@
+import { GetServerSidePropsContext } from 'next';
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client';
+import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { getSession } from 'next-auth/react';
+import { session } from '__tests__/fixtures/session';
+import TestRouter from '__tests__/util/TestRouter';
+import { GqlMockedProvider } from '__tests__/util/graphqlMocking';
+import makeSsrClient from 'src/lib/apollo/ssrClient';
+import AccountPage, { getServerSideProps } from './account.page';
+
+jest.mock('src/lib/apollo/ssrClient');
+
+const push = jest.fn();
+const router = {
+  push,
+};
+
+const context = {
+  req: {},
+} as unknown as GetServerSidePropsContext;
+
+describe('Setup account page', () => {
+  it('renders account options and saves default account renders and button saves and advances to the next page', async () => {
```

Comment:
Can you add a to make sure the mutation doesn't go off when no default account list is selected?

Reply:
I added a test that the save button is disabled until an account list is selected. Does that seem sufficient to you?

Reply:
That's fine